.github/ISSUE_TEMPLATE/formatting.md

```diff
@@ -0,0 +1,44 @@
+---
+name: ✨ Formatting
+about: Issues for ugly or incorrect code
+---
+
+<!--
+
+BEFORE SUBMITTING AN ISSUE:
+
+1.  Search for your issue on GitHub: https://github.com/prettier/plugin-php/issues
+    A large number of opened issues are duplicates of existing issues.
+    If someone has already opened an issue for what you are experiencing,
+    you do not need to open a new issue — please add a 👍 reaction to the
+    existing issue instead.
+
+2.  We get a lot of requests for adding options, but Prettier is
+    built on the principle of being opinionated about code formatting.
+    This means we have a very high bar for adding new options.
+    Find out more: https://prettier.io/docs/en/option-philosophy.html
+
+-->
+
+**Prettier 1.14.2**
+
+**PHP Plugin 0.6.0**
```

Comment:
I think it is good to request `prettier` version also, because some bugs can be in prettier